neural_speed/__init__.py

```diff
@@ -95,7 +95,10 @@ def _get_model_type(model_config):
     if model_type == "chatglm" and "chatglm3" in model_config._name_or_path:
         # due to the same model architecture.
         model_type = "chatglm2"
-
+    # For GLM4
+    if model_type == "chatglm" and "glm-4" in model_config._name_or_path:
+        # due to the same model architecture.
+        model_type = "chatglm2"
```

Comment:
Please also update this `model_type` in https://github.com/intel/neural-speed/blob/main/scripts/huggingface.py#L401. Otherwise, it will breke the ACC test for `GLM4`

Reply:
ok, thanks, god Tao!